content/user/troubleshooting/new_php_debug_logs.md

```diff
@@ -0,0 +1,76 @@
+---
+title: New PHP Debug Logs 
+description: Newer PHP Debug Logs in Zen Cart 
```

Comment:
Does this duplicate `/user/troubleshooting/strict_error_reporting`?

Reply:
No - I'm trying to explain how things that used to work sometimes won't after a PHP update.

Reply:
Okay, if it's related to a PHP upgrade, then maybe the title can reflect that more? Perhaps something like "Troubleshooting errors after upgrading PHP versions"?